app/views/declaration/destinationCountries/routing_countries_summary.scala.html

```diff
@@ -44,9 +45,14 @@
             @countriesOfRouting.zipWithIndex.map { case (country, index) =>
                 <tr class="table-row">
                     <th class="table-head" id="country_code_@index">@messages("declaration.routingCountries.summary.table.code")</th>
-                    <td class="table-cell" id="country_@index">@country</td>
+                    <td class="table-cell" id="country_@index">@country.toString()</td>
```

Comment:
please use `asString` or `asText` to distinguish from `Object#toString`

Reply:
Changed to `asString`